app/services/onboarding_csv_reader.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+#--------------------------
+#
+# @class OnboardingCsvReader
+#
+# @desc Responsibility: Import Data in an Onboarding CSV file.  Create
+#   objects as needed if they don't already exist.
+#
+# @author Ashley Engelund (ashley.engelund@gmail.com  weedySeaDragon @ github)
+# @date   11/01/20
+#
+#--------------------------
+#
+class OnboardingCsvReader
+  IMPORT_FN = Rails.root.join('onboarding_data.csv')
+
+  def self.import(full_filename = IMPORT_FN)
```

Comment:
I wouldn't set a default here, and I'd remove `onboarding_data.csv` from the file directory; it should accept a file as a param, and exit early/catch an error if one isn't provided.  I'll be running these data imports as individual rake tasks for unique data files every time we have to import one.